zcash_primitives/src/consensus.rs

```diff
@@ -240,7 +240,7 @@ impl Parameters for TestNetwork {
             NetworkUpgrade::Blossom => Some(BlockHeight(584_000)),
             NetworkUpgrade::Heartwood => Some(BlockHeight(903_800)),
             NetworkUpgrade::Canopy => Some(BlockHeight(1_028_500)),
-            NetworkUpgrade::Nu5 => Some(BlockHeight(1_599_200)),
+            NetworkUpgrade::Nu5 => None,
```

Comment:
This is unset for now because this height doesn't apply to the new consensus branch ID. We will set it to the actual height just before 4.7.0.